server/src/server.ts

```diff
@@ -172,46 +172,7 @@ function isPolisDev(uid?: any) {
   console.log("polisDevs", polisDevs);
   return polisDevs.indexOf(uid) >= 0;
 }
-// log heap stats
-setInterval(function () {
-  let mem = process.memoryUsage();
-  let heapUsed = mem.heapUsed;
-  let rss = mem.rss;
-  let heapTotal = mem.heapTotal;
-  winston.log(
-    "info",
-    "heapUsed:",
-    heapUsed,
-    "heapTotal:",
-    heapTotal,
-    "rss:",
-    rss
-  );
-  // let start = Date.now();
-
-  //metric("api.process.mem.heapUsed", heapUsed, start);
-  //metric("api.process.mem.rss", rss, start);
-  //metric("api.process.mem.heapTotal", heapTotal, start);
-}, 10 * 1000);
```

Comment:
Going to create an issue for this; There may have been a reason that we were specifically tracking heap usage here. We should look at how this might be more easily tracked in whatever platform we end up deploying on. For now, I think ok to delete, just want to flag so we can come back to it.